index.md

```diff
@@ -108,11 +108,11 @@ If you do not already have an Istio-enabled cluster but want to see what Kiali i
 
 ```
 wget https://raw.githubusercontent.com/kiali/kiali/master/hack/cluster-openshift.sh
-sh cluster-openshift.sh --istio-version 0.7.1 --kiali-enabled true up
+sh cluster-openshift.sh --istio-version 0.8 --kiali-enabled true up
```

Comment:
I don't think this is right. I believe you need to specify an exact, specific version - 0.8.0 for example.